src/SignalRServiceExtension/Bindings/SignalRInputBindings/InputBindingProvider.cs

```diff
@@ -29,10 +33,9 @@ public Task<IBinding> TryCreateAsync(BindingProviderContext context)
             {
                 switch (attr)
```

Comment:
Seems you can't get `GetCustomAttributes` from non-C# language. Thus, you will get a `null` to make the binding failed. Did you test in js?

Reply:
Tested, works. It's an inner provider, the outer provider is in webjob sdk, we don't need to handle it.

Reply:
will support it later